README.md

```diff
@@ -1 +1,50 @@
+# GSQ Results Profile
+This profile describes the `Results` concept.
```

Comment:
should have description similar to the other SOSA profiles
Obs = An observation is the act of carrying out an observation using a procedure to estimate or calculate a value of a geological or administrative feature.
Result ? A result is a quantitative or qualitative value of a defined attribute of a geological or administrative feature. that is estimated or calculated by the act of observation.